setup.cfg

```diff
@@ -1,3 +1,64 @@
+[metadata]
+name = pip
+author = The pip developers
+author_email = pypa-dev@groups.google.com
+license = MIT
+license_file = LICENSE.txt
+description = The PyPA recommended tool for installing Python packages.
+keywords =
+    distutils
+    easy_install
+    egg
+    setuptools
+    wheel
+    virtualenv
+url = https://pip.pypa.io/
+long_description = file: README.rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Topic :: Software Development :: Build Tools
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+
+project_urls =
+    Documentation = https://pip.pypa.io
+    Source = https://github.com/pypa/pip
+    Issues = https://github.com/pypa/pip/issues
+    Travis CI = https://travis-ci.org/pypa/pip
```

Comment:
There's also gh actions and azure

Reply:
I'd rather not add these additional links - if someone wants to do that, please make a follow up PR.

Reply:
GH actions are visible to signed-in accounts only. So I cannot recommend them. IDK about azure pipelines, I prefer to stay away from M$ inherent services not working without JS (including Windows Update, fortunately for the case of Windows Update the needed actions can be done manually).